Kconfig.zephyr

```diff
@@ -757,6 +757,9 @@ config CHECK_INIT_PRIORITIES
 	  priority), see CHECK_INIT_PRIORITIES_FAIL_ON_WARNING to fail on
 	  warnings (dependent devices, same priority) as well.
 
+	  NOTE: this check is does not support armclang binaries and is
+	  currently silently skipped for those.
+
```

Comment:
if adding dependency on setting, then this note can be removed.
```suggestion
```

Reply:
Yes! done.